src/js/modules/infragistics.ui.tree.js

```diff
@@ -747,7 +747,7 @@
 				//Set
 				settings.allowDrop = {
 					allowDrop: false,
-					```
+					...
```

Comment:
Actually only the overview code snippet is copy-paste able by our current conventions. When we leave "..." in initialize code snippet we mean there might be other options. We can replace the "..." with "//Other options go here", which will make it copy-paste able, but in that case we'll need to update most of the option's code snippets.